### ISSUE TITLE
FeedbackProfile: Question, Scale and QuestionScale entity property conflicts

### ISSUE THREAD

**Issue by arwhyte** [IMSGlobal/caliper-spec#379]
The latest merge added `Question` and a number of question subtypes.  It also introduced the following issues:

* `Question` subtype of `DigitalResource`
  - `Question.questionPosed<string>`
* `Scale` subtype of `Entity`
  - `Scale.question<string>`  <-- WRONG TYPE
* ScaleQuestion subtype of `Question` and `Scale`
  -  `ScaleQuestion.questionPosed<string>`
  - `ScaleQuestion.question<string>`   <-- REDUNDANT
* `Rating` subtype of `Entity`
  - `Rating.scaleQuestion<ScaleQuestion>`

Change to:

* `Question` subtype of `DigitalResource`
  - `Question.questionPosed<string>`
* `Scale` subtype of `Entity`
  - `Scale.question<Question>`  <-- Question entity not string
* ScaleQuestion subtype of `Question` and `Scale` <-- DELETE ScaleQuestion as redundant
* `Rating` subtype of `Entity`
  - `Rating.scale<Scale>` <-- REVERT

In short, have `Scale` reference `Question`.

An alternative:

* `Question` subtype of `DigitalResource`
  - `Question.questionPosed<string>`
* `Scale` subtype of `Entity`
  - `Scale.question<Question>`  <-- DELETE
* `Rating` subtype of `Entity`
  - `Rating.scale<Scale>` <-- REVERT
  -  `Rating.question<Question>`

An alternative

* `Question` subtype of `DigitalResource`
  - `Question.questionPosed<string>`
* `Scale` subtype of `Entity`
  - `Scale.question<Question>`  <-- DELETE
* `RatingScaleQuestion` subtype of `Question`
  - `RatingScaleQuestion.scale<Scale>`
* `Rating` subtype of `Entity`
  -  `Rating.question<Question>`

**Comment by arwhyte** [comment]
Resolved by #380.
